src/backend/distributed/transaction/transaction_recovery.c

```diff
@@ -474,7 +474,7 @@ PendingWorkerTransactionList(MultiConnection *connection)
 	int32 coordinatorId = GetLocalGroupId();
 
 	appendStringInfo(command, "SELECT gid FROM pg_prepared_xacts "
-							  "WHERE gid LIKE 'citus\\_%d\\_%%' and database = current_database()",
```

Comment:
Any idea why `%%` is used here? Wouldn't `%` do the same? We could fix it while we are here